lib/js/components/Headers/SectionHeader/SectionHeader.vue

```diff
@@ -0,0 +1,343 @@
+<template>
+	<div
+		class="sectionHeader"
+		:class="{
+			'-expandable': expandable,
+			'-horizontal': mobileLayout === SECTION_HEADER_MOBILE_LAYOUTS.HORIZONTAL,
+			[sizeClass]: true,
+		}"
+	>
+		<div class="sectionHeader__wrapper">
```

Comment:
can we merge `secrionHeader` and `__wrapper`. I see there is a divider but `__wrapper` has `flex-direction: column` anyway.